examples/network_profile/isolation_with_external_subnet/main.tf

```diff
@@ -0,0 +1,36 @@
+provider "vra" {
+  url           = var.url
+  refresh_token = var.refresh_token
+}
+
+data "vra_cloud_account_aws" aws-cloud-account {
+       name = var.cloud_account
+}
+
+data "vra_region" "my-region" {
```

Comment:
This is a general comment and something I likely introduced originally. We should think about removing the use of "my" in these examples and introduce a new variable (var.name?) that gets set in the invocation. That would make it easier to have multiple people use these as templates without conflicts with existing infrastructure. Thoughts?

Reply:
I'm all up for removing "my" from the examples. But not sure if it is really necessary to introduce another variable. By "conflicts with existing infrastructure", do you mean the actual infrastructure that is being managed by these manifests or the resource/data-source blocks in the manifests?